project_euler/problem_102/sol1.py

```diff
@@ -0,0 +1,83 @@
+"""
+Three distinct points are plotted at random on a Cartesian plane,
+for which -1000 ≤ x, y ≤ 1000, such that a triangle is formed.
+
+Consider the following two triangles:
+
+A(-340,495), B(-153,-910), C(835,-947)
+
+X(-175,41), Y(-421,-714), Z(574,-645)
+
+It can be verified that triangle ABC contains the origin, whereas
+triangle XYZ does not.
+
+Using triangles.txt (right click and 'Save Link/Target As...'), a 27K text
+file containing the coordinates of one thousand "random" triangles, find
+the number of triangles for which the interior contains the origin.
+
+NOTE: The first two examples in the file represent the triangles in the
+example given above.
+"""
+
+from pathlib import Path
+from typing import List, Tuple
+
+vector = Tuple[int, int]
```

Comment:
Hey, I fixed this for the bot. Now if the assignment is for a type alias, the bot won't complain.
Please change the name to CamelCase and also end it with a suffix `T` or `Type` to denote it as a type alias: `VectorT` or `VectorType`

Reply:
Also, I don't think a point or coordinate represents a `Vector`. 

Another tip: Use type alias only if it is complex nested types and multiple return variables from a function (as a tuple). I will leave it up to you to decide whether we need a type alias and what the name should be.